site/src/jsMain/kotlin/com/zenmo/web/zenmo/components/widgets/MediaContentLayout.kt

```diff
@@ -0,0 +1,207 @@
+import com.varabyte.kobweb.silk.style.extendedBy
+import com.varabyte.kobweb.silk.style.toModifier
+import com.varabyte.kobweb.silk.theme.breakpoint.rememberBreakpoint
+import com.zenmo.web.zenmo.domains.lux.widgets.headings.HeaderText
+import com.zenmo.web.zenmo.domains.lux.widgets.headings.SubHeaderText
+import com.zenmo.web.zenmo.domains.zenmo.widgets.button.PrimaryButton
+import com.zenmo.web.zenmo.theme.font.TextStyle
+import com.zenmo.web.zenmo.theme.font.TextStyleBrandColor
+import com.zenmo.web.zenmo.theme.styles.OutlinePrimaryButtonStyle
+import org.jetbrains.compose.web.css.*
+import org.jetbrains.compose.web.dom.Div
+import org.jetbrains.compose.web.dom.P
+
+val MediaContentDisplayStyle = CssStyle {
+    base {
+        Modifier
+            .display(DisplayStyle.Flex)
+    }
+}
+
+val MediaContentLayoutStyle = MediaContentDisplayStyle.extendedBy {
+    Breakpoint.ZERO {
+        Modifier.flexDirection(FlexDirection.ColumnReverse)
+    }
+    Breakpoint.SM {
+        Modifier.flexDirection(FlexDirection.ColumnReverse)
+    }
+    Breakpoint.MD {
+        Modifier.flexDirection(FlexDirection.Row)
+    }
+    Breakpoint.LG {
+        Modifier.flexDirection(FlexDirection.Row)
+            .width(90.percent)
+    }
+    Breakpoint.XL {
+        Modifier
+            .flexDirection(FlexDirection.Row)
+            .width(85.percent)
+    }
+}
+
+val ReversedMediaContentLayoutStyle = MediaContentDisplayStyle.extendedBy {
+    Breakpoint.ZERO {
+        Modifier.flexDirection(FlexDirection.Column)
+    }
+    Breakpoint.SM {
+        Modifier.flexDirection(FlexDirection.Column)
+    }
+    Breakpoint.MD {
+        Modifier.flexDirection(FlexDirection.RowReverse)
+    }
+    Breakpoint.LG {
+        Modifier.flexDirection(FlexDirection.RowReverse)
+    }
+    Breakpoint.XL {
+        Modifier.flexDirection(FlexDirection.RowReverse)
+    }
+}
+
+
+@Composable
+fun MediaContentLayout(
+    enTitle: String,
+    nlTitle: String,
+    enDescription: String,
+    nlDescription: String,
+    imageUrl: String,
+    enSubtitle: String? = null,
+    nlSubtitle: String? = null,
+    mediaAction: (() -> Unit)? = null,
+    enActionText: String? = null,
+    nlActionText: String? = null,
+    reversed: Boolean = false,
+) {
+    val breakpoint = rememberBreakpoint()
+    val divModifier = if (reversed) {
+        ReversedMediaContentLayoutStyle.toModifier()
+    } else {
+        MediaContentLayoutStyle.toModifier()
+    }
+    Div(
+        attrs =
+            divModifier
+                .gap(if (breakpoint < Breakpoint.MD) 2.cssRem else 5.cssRem)
+                .fillMaxWidth()
+                .alignItems(if (breakpoint < Breakpoint.MD) AlignItems.Start else AlignItems.Center)
+                .toAttrs()
+    ) {
+        ImageContent(imageUrl)
+        Column(
+            horizontalAlignment = Alignment.Start,
+            modifier = Modifier
+                .fillMaxWidth()
+        ) {
+            TextContent(
```

Comment:
On some device widths the content is overflowing the column on the right.

Reply:
Hmm this is because of the long word in the title. 

Maybe the header font is too big and/or the image is too wide.